src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -156,7 +156,7 @@ public void testModerateResponsesButton() {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByQuestion();
-        ThreadHelper.waitFor(2000);
+        resultsPage.waitForPageToLoad();
```

Comment:
Thanks for removing this! But I think <code>displayByQuestion</code> already performs a waitForPageToLoad?